only considering 1 layer (#18)
SpecialeBajs/speciale-rapport: article/Experiment/experiment.tex

\section{Experiment}
\textbf{RQ1}: How does GCF perform when removing parts of the embedding propagation?\\

> \textbf{RQ2}: How does LightGCN perform the final embedding is a certain convolution layer? \\

Reviewer: ```suggestion
\textbf{RQ2}: How does LightGCN perform when only one convolution layer is used for the final embedding ? \\
```
